src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -185,7 +185,9 @@ namespace ReportSizingManager {
 
 		// add to SQL output
 		if ( sqlite ) sqlite->addSQLiteComponentSizingRecord( CompType, CompName, VarDesc, VarValue );
-
+		if ( present( UsrDesc ) && present( UsrValue ) ) {
+			if ( sqlite ) sqlite->addSQLiteComponentSizingRecord( CompType, CompName, UsrDesc, UsrValue );
```

Comment:
This works as expected, for most files it will results in many more entries for this table in sql when design size and user-specified values are present.  For example file 5ZoneAutoDXVAV, this table in sql grows from 41 rows to 53 rows.  Checking with @shorowit and @macumber to be sure this won't break things that read this output.